.github/workflows/release-e2e-workflow-template-windows.yml

```diff
@@ -0,0 +1,138 @@
+name: Release tests workflow in Bundled OpenSearch Dashboards (Windows)
+on:
+  workflow_call:
+    inputs:
+      test-name:
+        required: true
+        type: string
+      test-command:
+        required: true
+        type: string
+      security-enabled:
+        required: false
+        type: string
+jobs:
+  tests:
+    name: Run Cypress E2E tests for ${{ inputs.test-name }} (Windows)
+    runs-on: windows-latest
+    env:
+      # prevents extra Cypress installation progress messages
+      CI: 1
+      # avoid warnings like "tput: No value for $TERM and no -T specified"
+      TERM: xterm
+    steps:
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
```

Comment:
Use 11 for 1.x right here, and 2.x+ should use 17 from now on.

Reply:
Since the workflow is only running against artifact, the java configuration is not relevant and shall be removed.